README.md

```diff
@@ -27,8 +27,7 @@ process that requires governance intervention to work around.
 In the latter case, governance can choose to refund the ESM pledgers by minting new
 tokens.
 
-If governance wants to disarm the ESM, it can only do so by removing its
-authorization to call `end.cage()`.
+If governance wants to disarm the ESM, it can do so by removing all of it's authorizations in the system or by calling `cage()`.
```

Comment:
nit:  I think this is a good backstop so that we know that an ESM can't fire, but I feel pretty strongly that we should de-auth it in the system as well.  I don't like the idea of dead auths hanging around.

Reply:
```suggestion
If governance wants to disarm the ESM, it can do so by removing all of it's authorizations in the system and then by by calling `cage()` on this contract.
```

Reply:
Yes I agree and had that in the back of my head. Happy to enshrine it in the README.